packages/types/src/promotion/service.ts

```diff
@@ -0,0 +1,5 @@
+import { ModuleJoinerConfig } from "../modules-sdk"
+
+export interface IPromotionModuleService {
```

Comment:
todo: In the search we have add introduce a bsae interface such as 
```ts
export interface IModuleService {
  __joinerConfig?(): ModuleJoinerConfig
}
```
The idea is that all modules service interface extend it so we centralize those common properties and we can add more in the future such as `_hooks` without having to change all the other interfaces. Can I have you to add that please